build_image.sh

```diff
@@ -101,7 +101,7 @@ _output_file(){
         fi
         if _file_exists $app_b_bin
         then
-            dd if=$app_b_bin of="$OUT_BIN" bs=1 count=$APP_B_SIZE seek=$APP_B_OFFSET
+            dd if=$app_b_bin of="$OUT_BIN" bs=1 count=$APP_B_SIZE seek=$APP_B_OFFSET conv=notrunc
```

Comment:
@Cezarus27 why it is necessary to make such change?

Reply:
Without it generated image isn't 1MB fixed. Codewallet signing mechanism required file of fixed 1MB size.

Reply:
Ok

Reply:
@Cezarus27 let's add here some comment for the future why we use `notrunc`

Reply:
@TomaszAIR Okey, I added the comment inside the script.